exchange/utils.go

```diff
@@ -442,11 +442,18 @@ func buildRequestExtForBidder(bidder string, requestExt json.RawMessage, request
 }
 
 func buildRequestExtAlternateBidderCodes(bidder string, accABC *openrtb_ext.ExtAlternateBidderCodes, reqABC *openrtb_ext.ExtAlternateBidderCodes) *openrtb_ext.ExtAlternateBidderCodes {
+	if len(bidder) == 0 {
+		return nil
+	}
+
+	normalisedBidderName, _ := openrtb_ext.NormalizeBidderName(bidder)
+	ciBidder := string(normalisedBidderName)
```

Comment:
Could we change the variable name of `ciBidder` to `caseInsensitiveBidder` or something a bit more descriptive?

Reply:
Done